carma/launch/drivers.launch.py

```diff
@@ -102,10 +110,23 @@ def generate_launch_description():
         arguments=['--ros-args', '--log-level', GetLogLevel('subsystem_controllers', env_log_levels)]
     )
 
+    # subsystem_controller which orchestrates the lifecycle of this subsystem's components
+    mock_controller_driver = Node(
+        package='mock_controller_driver',
+        name='mock_controller_driver',
+        executable='mock_controller_driver_node_exec',
+        parameters=[
+            {"use_mock_controller" : use_mock_controller}],
+        arguments=['--ros-args', '--log-level', GetLogLevel('mock_controller_driver', env_log_levels)],
+        condition=IfCondition(LaunchConfiguration('use_mock_controller'))
+    )
+
     return LaunchDescription([
```

Comment:
Actually maybe we should not even add a call here because we normally call it form a different drivers.launch.py in carma-config/development.

Reply:
I agree, removing the parameter and launch description.